src/components/form/FormContainer.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { Heading, Paragraph } from '../shared';
+import Inputs from '../shared/partials/FormInputs';
+import formimg from '../../assets/images/form-tickets.png';
+import { feeCoverage, initialContent, inputContent, lastPartContent } from '../../data/formInformation';
+
+const FormContainer = () => {
```

Comment:
Add a state which tracks the whole form state. 
Reference :- https://github.com/dscnitrourkela/project-raisin/blob/2abf016b6937f9eebe8faefc3027c45321d3f460/src/components/Registration/RegistrationContext.jsx#L34-L59

Also add a setInputValue like [this](https://github.com/dscnitrourkela/project-raisin/blob/2abf016b6937f9eebe8faefc3027c45321d3f460/src/components/Registration/RegistrationContext.jsx#L94-L104)

Reply:
done . https://github.com/dscnitrourkela/project-udon/pull/11/commits/8e2296e1ce042b395989b5288cbea5961793066f